site_scons/env_modules.py

```diff
@@ -39,14 +39,10 @@ def __init__(self):
         """Load Modules for initializing envirables"""
         # Leap 15's lmod-lua doesn't include the usual module path
         # in it's MODULEPATH, for some unknown reason
-        os.environ["MODULEPATH"] = ":".join([os.path.sep +
-                                             os.path.join("usr", "share",
-                                                          "modules"),
-                                             os.path.sep +
-                                             os.path.join("etc",
-                                                          "modulefiles")] +
-                                            os.environ.get("MODULEPATH",
-                                                           "").split(":"))
+        os.environ["MODULEPATH"] = ":".join([os.path.join("/usr", "share", "modules"),
```

Comment:
This version worked but I had also updated it since to use os.sep

Reply:
Arghhh, I did not check my cherry pick :'(

Reply:
Going to fix it asap

Reply:
Fixed and checked with your branch.
My apologies.